File: domein/src/main/java/be/genealogie/domein/dto/RelatieMetNieuwNatuurlijkPersoonDto.java
package be.genealogie.domein.dto;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDate;

@Data
@Builder
public class RelatieMetNieuwNatuurlijkPersoonDto {
    private Long id;
    private NatuurlijkPersoonDTO persoon1;
    private NieuwNatuurlijkPersoonDTO persoon2;
    private LocalDate gehuwdOp;
    private String gehuwdTe;
}

File: frontend/src/app/model/relatie-met-nieuw-natuurlijk-persoon-dto.ts
import {NatuurlijkPersoonDto} from "./natuurlijk-persoon-dto";

export class RelatieMetNieuwNatuurlijkPersoonDto {
  public constructor(
    public persoon1:NatuurlijkPersoonDto | null | undefined,
    public persoon2:NatuurlijkPersoonDto,
    public gehuwdOp:Date | null,
    public gehuwdTe:string | null,

  ) {
  }
}

